activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Fix ActiveRecord query attribute method when given value does't respond to to_i method
```

Comment:
This PR will make that query attribute method on user defined attribute awares type casted value.

For example, on `attribute :user_defined_text, :text`, `klass.new(user_defined_text: "false").user_defined_text?` no longer return false as casted non-empty string.

So the chengelog entry should mention about user defined attribute.

```suggestion
*   Fix query attribute method on user defined attribute to aware type casted value.
```

Reply:
Appreciate for your excellent detailed review!!!